.github/workflows/image-update.yaml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v1
-      - run: make -f Makefile.docker eksctl-image
       - run: make -f Makefile.docker check-build-image-manifest-up-to-date || (make -f Makefile.docker update-build-image-manifest && make -f Makefile.docker push-build-image)
```

Comment:
So this line is misleading right now, it should be just `make -f Makefile.docker check-build-image-manifest-up-to-date` for now, the rest is currently a wishlist.

Reply:
Not at the moment, I have other stuff to do. I just want the ci jobs to start working again :/